pkg/api/dashboard.go

```diff
@@ -470,14 +470,20 @@ func (hs *HTTPServer) postDashboard(c *models.ReqContext, cmd models.SaveDashboa
 		}
 
 		if liveerr != nil {
-			hs.log.Warn("unable to broadcast save event", "uid", dashboard.Uid, "error", err)
+			hs.log.Warn("unable to broadcast save event", "uid", dashboard.Uid, "error", liveerr)
```

Comment:
Feel free to ignore this comment.

I really don't like this variable name, since it reminds me of the human organ `liver` rather than and error from `live`.

```suggestion
			hs.log.Warn("unable to broadcast save event", "uid", dashboard.Uid, "error", liveErr)
```

Reply:
Haha, that's pretty funny :) I'll leave it as it is for now, as I didn't introduce this error variable and don't want to add too many changes to this PR. But I'll keep it in mind for later.